src/Tribe/Views/V2/View.php

```diff
@@ -1045,7 +1038,7 @@ protected function setup_template_vars() {
 				'compact'        => Dates::datepicker_formats( tribe_get_option( 'datepickerFormat' ) ),
 				'month_and_year' => tribe_get_date_option( 'monthAndYearFormat', 'F Y' ),
 			],
-			'messages'           => $this->get_messages(),
+			'messages'           => $this->get_messages( $events ),
```

Comment:
:warning: Array double arrow not aligned correctly; expected 10 space(s) between "'messages'" and double arrow, but found 11.

<sub>hash: 9aee459749d88d2cb7c0f2d87c038502</sub>